exercises/concept/phone-number-analysis/.docs/instructions.md

```diff
@@ -0,0 +1,32 @@
+# Instructions
+
+This exercise has you analyze phone numbers.
+
+You are asked to implement 2 features.
+
+Phone numbers passed to the routines are guaranteed to be in the form
+NNN-NNN-NNNN e.g. 212-515-9876.
+
+## 1. Analyze a phone number
+
+Your analysis should return 3 pieces of data
+
+1. An indication of whether the number has a New York dialing code ie. 212 as the first 3 digits
+2. An indication of whether the number is fake having 555 as a prefix code in positions 5 to 7 (numbering from 1)
+3. The last 4 digits of the number.
+
+Implement the function `analyze()` to produce the phone number info.
+
+```rust
+analyze("631-555-1234");
+// => (false, true, "1234")
+```
+
+## 2. Detect if a phone number has a fake prefix code (555)
```

Comment:
Many people would consider that feature 2 would be a standalone test and that feature 1 would reference that test in constructing the feature 1 tuple. It seems that we are looking for feature 2 to return the second element of the feature 1 tuple (i.e. analyze().1) , rather than returning the value of the separate test, simply because the second part of feature 1 relies on feature 2.

Perhaps we should recognize that in the instructions and say (as we do in different exercises) that although you could simply return the value of the standalone test for fake 555 prefix, this exercise asks you to return the value based only on the phone number info produced in task 1.